### ISSUE TITLE
Automate regression testing

### ISSUE THREAD

**Issue by havijw** [EasyArray/phosphorus#25]
Using something like [nbval](https://github.com/computationalmodelling/nbval) to do automatic regression testing would be a lot faster and more precise than manually running through notebooks and checking the output.

I can look into this and see how easy and appropriate this would be for our purposes.

**Comment by EasyArray** [comment]
Yes, thanks! Good idea. It compares the output to previous outputs, so we'd just have to update it when we change what outputs look like or do...

**Comment by EasyArray** [comment]
Also relevant: https://stackoverflow.com/questions/40172281/unit-tests-for-functions-in-a-jupyter-notebook
